bacula_fd_operator/src/bacula.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""A helper library for managing bacula-fd"""
+
+from pathlib import Path
+
+import jinja2
+
+import charms.operator_libs_linux.v0.apt as apt
+import charms.operator_libs_linux.v0.systemd as systemd
+
+BACULA_FD_CONFIG_TEMPLATE_FILE = Path(__file__).parent / "templates/bacula-fd.conf.j2"
+BACULA_FD_CONFIG_FILE = Path("/etc/bacula/bacula-fd.conf")
+
+
+def is_installed() -> bool:
+    """Check if bacula-fd is installed.
+
+    Returns:
+        True if bacula-fd is installed.
+    """
+    return Path("/usr/sbin/bacula-fd").exists()
```

Comment:
This won't be resilient if the bacula deb changes the default folder
Isn't it more reliable to check if the `bacula-fd` deb is installed ?